app/data/etl/outdated_items.rb

```diff
@@ -39,7 +39,7 @@ def create_new_items(items)
   def import_content_details(items)
     log process: :import_content_details, message: "creating #{items.length} ImportContentDetailsJobs"
     items.each do |item|
-      ImportContentDetailsJob.perform_async(item.content_id, item.base_path)
+      ImportContentDetailsJob.perform_async(item.content_id, item.base_path, item.locale)
```

Comment:
What about passing only the `item.id`? We are passing all these parameters, just to be able to find the right item.